src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -42,7 +42,7 @@ export const GithubCreateBranchPost = async (req: Request, res: Response): Promi
 		sendTrackEventAnalytics(AnalyticsTrackEventsEnum.CreateBranchSuccessTrackEventName, jiraHost);
 	} catch (err) {
 		req.log.error({ err }, "Error creating branch");
-		res.sendStatus(500);
+		res.status(err.status).json(errorMessages[err?.status || 500]);
```

Comment:
Since we use `err?.status || 500` in the res body, should we also use `err?.status || 500` in the `res.status` call as well?